plugins/woocommerce-admin/client/wp-admin-scripts/order-tracking/exit-order-page.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * Internal dependencies
+ */
+import { addCustomerEffortScoreExitPageListener } from '~/customer-effort-score-tracks/customer-effort-score-exit-page';
+import { staticFormDataToObject } from '~/utils/static-form-helper';
+
+type FormElements = {
+	post?: HTMLFormElement;
+} & HTMLCollectionOf< HTMLFormElement >;
+const forms: FormElements = document.forms;
+if ( forms && forms.post ) {
```

Comment:
Is there a reason not to do `if( forms?.post ) {` here?

Reply:
No, that seems like a better approach, thanks!